sky/provision/aws/instance.py

```diff
@@ -598,14 +598,13 @@ def open_ports(
                                   excluded_instances=None)
     instance_list = list(instances)
     if not instance_list:
-        logger.warning(
-            f'Instance with cluster name {cluster_name_on_cloud} not found. '
-            f'Skip creating security group.')
-        return None
+        raise ValueError(
```

Comment:
Should we add `with ux_utils.print_exception_no_traceback`?

Reply:
Done. Thanks!